substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/throttling/JfrEventThrottler.java

```diff
@@ -47,9 +48,11 @@ public class JfrEventThrottler extends JfrAdaptiveSampler {
     private static final long TEN_PER_1000_MS_IN_HOURS = 36000;
     private static final long DAY = 24 * HOUR;
     private static final long TEN_PER_1000_MS_IN_DAYS = 864000;
-
+    @BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-23+8/src/hotspot/share/jfr/recorder/service/jfrEventThrottler.cpp#L105-L106") //
     private static final long DEFAULT_WINDOW_LOOKBACK_COUNT = 25;
+    @BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-23+8/src/hotspot/share/jfr/recorder/service/jfrEventThrottler.cpp#L112-L113") //
```

Comment:
This and the other two annotations for the constant above and below should use a single line reference. Currently they span two lines, see https://github.com/openjdk/jdk/blob/jdk-23+8/src/hotspot/share/jfr/recorder/service/jfrEventThrottler.cpp#L112-L113.

```suggestion
    @BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-23+8/src/hotspot/share/jfr/recorder/service/jfrEventThrottler.cpp#L112") //
```

Other annotations that should reference a single source line but use a range are probably affected as well.

Reply:
Ok I've updated the relevant annotations to point to a single line instead of a range. [I needed up update the regex here too.](https://github.com/oracle/graal/pull/8542/files#diff-8fcc9f49f232b75e70027a8ec6252e86ace325eb4cb9010238885b47bc49295cR76)

Reply:
ah right. apparently, we did not yet use single line references. thanks for fixing.